site/assets/scss/_variables_project.scss

```diff
@@ -3,16 +3,17 @@
 // minikube colors, taken from the logo
 $mk-dark: #306EE5;
 $mk-medium: #1FC3CF;
+$mk-midlight: #e8f3f3;
 $mk-light:#C7EAEC;
 $mk-verydark: darken($mk-dark, 15%);
 
 // bootstrap colors
 $dark: #403F4C !default;
 $blue: #72A1E5 !default;
 $orange: #BA5A31 !default;
-$gray-100: #f8f9fa !default;
+$gray-100: #f6f6f8 !default;
```

Comment:
the new gray is a bit too dark, maybe just little bit shad of lighter (I am reffering to the background of the tabs)  not sure which one of the grays is used there

Reply:
I made it a lighter form of minikube-blue again. How's it look?